--- mimalloc-2.2.3/CMakeLists.txt.orig	2025-03-28 22:19:07.000000000 +0100
+++ mimalloc-2.2.3/CMakeLists.txt	2025-04-27 08:27:46.785275340 +0200
@@ -534,8 +534,6 @@ string(TOLOWER "${CMAKE_BUILD_TYPE}" CMA
 list(APPEND mi_defines "MI_CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE_LC}")  #todo: multi-config project needs $<CONFIG> ?
 if(CMAKE_BUILD_TYPE_LC MATCHES "^(release|relwithdebinfo|minsizerel|none)$")
   list(APPEND mi_defines MI_BUILD_RELEASE)
-else()
-  set(mi_libname "${mi_libname}-${CMAKE_BUILD_TYPE_LC}") #append build type (e.g. -debug) if not a release version
 endif()
 
 if(MI_BUILD_SHARED)
